modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/entity/v1_0/ObjectEntryEntityModel.java

```diff
@@ -158,7 +158,9 @@ else if (Objects.equals(objectField.getDBType(), "Long")) {
 						"nestedFieldArray.value_long#" +
 							objectField.getName()));
 		}
-		else if (Objects.equals(objectField.getDBType(), "String")) {
+		else if (Objects.equals(objectField.getDBType(), "Clob") ||
+				 Objects.equals(objectField.getDBType(), "String")) {
+
```

Comment:
I think we should reorder so that `(Objects.equals(objectField.getDBType(), "Clob")` is before `(Objects.equals(objectField.getDBType(), "Date"))`.
If it's my only observation, I don't mind sending it this way so we don't have to run the CI again 😄

Reply:
No problem. We run again 😉 
Thanks for the review, @carolmariaabb